### DISCUSSION TITLE
opt+key not working on macos. possible bug?

### DISCUSSION THREAD

**Discussion by hamzashezad** [wezterm/wezterm#4739] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20231228-084719-ff274374

### Ask your question!

No alternative keys; Opt + keys, are printed as expected, for example Opt+3 should print £ but it doesn't print anything. I have no configuration file. This seems like a basic requirement so opening a discussion before filing an issue. Am I missing something here?

**Comment by wez** [comment, marked as answer]
See https://wezfurlong.org/wezterm/config/keyboard-concepts.html#macos-left-and-right-option-key

**Reply by hamzashezad** [reply]
Got it, thanks!
